src/libraries/System.Text.Json/tests/System.Text.Json.SourceGeneration.Unit.Tests/JsonSourceGeneratorTests.cs

```diff
@@ -233,46 +219,16 @@ public sealed class JsonSerializableAttribute : JsonAttribute
             Assert.Empty(result.Diagnostics);
         }
 
-        [Theory]
-        [InlineData("System.Text.Json", true)]
-        [InlineData("System.Text.Json.Not", true)]
-        [InlineData("System.Text.Json", false)]
-        [InlineData("System.Text.Json.Not", false)]
-        public static void LocalJsonSerializableAttributeUnexpectedShape(string assemblyName, bool includeSTJ)
-        {
-            string source = """
-                using System;
-                using System.Text.Json.Serialization;
-
-                [assembly: JsonSerializable(typeof(int))]
```

Comment:
This (and a number of other unit tests) are calling early source gen APIs that were eventually removed before the SG was shipped.